cmd/zc_enumerator.go

```diff
@@ -85,8 +85,8 @@ func (storedObject *storedObject) isMoreRecentThan(storedObject2 storedObject) b
 }
 
 // a constructor is used so that in case the storedObject has to change, the callers would get a compilation error
-func newStoredObject(name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType, containerName string) storedObject {
-	return storedObject{
+func newStoredObject(morpher objectMorpher, name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType, containerName string) storedObject {
```

Comment:
Suggestion: Call it objectPreProcessor instead of objectMorpher

I feel as though either name is adequate, but preProcessor is the technical term you've used to refer to it in the past

Reply:
True. It is inconsistent.  BTW I almost made it consistent in the other direction: "morpher" everywhere, to try to convey the idea that it's _only_ job is to modify the storedObject; not to do anything outside that object.  In the interests of time, I'm going to leave the name as-is for now.